src/snippets.js

```diff
@@ -493,16 +493,16 @@ var SnippetManager = function() {
         tabstopManager.addTabstops(processedSnippet.tabstops, range.start, end, selectionId);
     };
     
-    this.insertSnippet = function(editor, snippetText, replaceRange) {
+    this.insertSnippet = function(editor, snippetText, replaceRange, removeExtraIndent) {
```

Comment:
can we make this feature flag kind of "removeExtraIndent" a config option in autocomplete or similar instead of passing it through functions?

Reply:
it needs to be passed per invocation to choose if indentation is needed, as snippets may or may not have indentation already